megaparse/sdk/megaparse_sdk/__init__.py

```diff
@@ -4,8 +4,8 @@
 
 
 class MegaParseSDK:
-    def __init__(self, api_key: str):
-        self.client = MegaParseClient(api_key)
+    def __init__(self, api_key: str, base_url: str | None = None, timeout: int = 600):
```

Comment:
Timeout won't affect the http request. By default in production, the timeout is on the laod balancer and not the app. It is at 60 seconds. Adding this here gives a false sense of: you can increase the timeout

Reply:
(and this is how I fixed it locally, but I understand that this is not a good solution for prod...)

Reply:
Agree with @StanGirard  👍🏼 . We should have faster parsing always  or send the parser a page by page of the pdf from the workers